src/foam/nanos/logger/DiscardLogger.js

```diff
@@ -0,0 +1,99 @@
+/**
+ * @license
+ * Copyright 2018 The FOAM Authors. All Rights Reserved.
+ * http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+foam.CLASS({
+  package: 'foam.nanos.logger',
+  name: 'DiscardLogger',
```

Comment:
Why is this called DiscardLogger instead of FilteredLogger and the boolean values not reversed to be enable instead of discard? This seems to be the reverse of normal?

Reply:
Makes sense, will amend accordingly.

Reply:
So there is another logger decorator in the pipeline, that will filter against regexs, and in the schema that was initially drawn out, that decorator was named FilteredLogger and the one implemented here DiscardLogger, hence the naming.
Flipped the boolean values.